app/controllers/public/sessions.js

```diff
@@ -17,6 +17,19 @@ export default class SessionsController extends Controller {
   preserveScrollPosition = true;
   my_schedule=null;
 
+  @computed('sort')
+  get sortTitle() {
+    switch (this.sort) {
+      case 'title':
+        return 'By Title';
+      case null:
+      case 'starts-at':
+        return 'By Time';
+      default:
+        return 'By Popularity';
+    }
+  }
+
   @computed('model.session.@each', 'timezone')
   get groupByDateSessions() {
     let sessions;
```

Comment:
line number 36. sir i am able to suggest changes to line no 36 thats why i am writing it here.
`    if (this.sort === 'title' || this.sort === '-favourite-count') {`

Sir this will work 
![2](https://user-images.githubusercontent.com/72552281/109898088-abea8700-7cb9-11eb-986f-3862b5e7f64b.png)